catalog/nsili/catalog-nsili-source/src/main/java/com/connexta/alliance/nsili/source/NsiliSource.java

```diff
@@ -152,6 +159,8 @@
 
     private static final String FILE = "file";
 
+    private static final int DEFAULT_NUM_WORKERS = 4;
```

Comment:
If the default is defined in metatype and blueprint, i don't think we need to set another default in code.